docs/src/concepts/index_ivfpq.md

```diff
@@ -0,0 +1,78 @@
+# Understanding LanceDB's IVF-PQ index
+
+An index is a data structure that represents data in a way that makes it more efficient to search and retrieve. Using an index is faster, but less accurate than kNN or brute force search because, in essence, the index is a lossy representation of the data.
```

Comment:
Might be worth specifying `ANN` index, since the part about being "less accurate" doesn't apply to other index types (like scalar indices).
```suggestion
An ANN (Approximate Nearest Neighbors) index is a data structure that represents data in a way that makes it more efficient to search and retrieve. Using an ANN index is faster, but less accurate than kNN or brute force search because, in essence, the index is a lossy representation of the data.
```